src/logics/LIALogic.h

```diff
@@ -32,6 +34,8 @@ class LIALogic: public LALogic
     static const char*  tk_int_plus;
     static const char*  tk_int_times;
     static const char*  tk_int_div;
+    static const char*  tk_int_mod;
+    static const char*  tk_int_abs;
```

Comment:
I noticed that `Int` theory declares also `abs` function, so I put it here, but maybe I can remove it since it is not necessary for what I need right now.